Improve using Radicle doc sections (#16)
radicle-dev/radicle-docs: docs/getting-started/getting-started.md

## Installation & Download

> The Radicle network can be installed and accessed via Upstream, a desktop client. To download the app, click [here](https://buildkite.com/monadic/radicle-upstream/builds/6098). While Upstream is currently the only client for using Radicle, it won't be the only one. Upstream development is public by default and completely open source. You can follow along development on [radicle.community](https://radicle.community)

Author: This sounds a bit like we're installing the radicle network on our machine, which to me doesn't make sense. What was wrong with the previous version?

Reviewer: Was trying to reflect the concept of "Radicle installation" that @xla brought up, but you're right, doesn't make sense with the subject. Will revert back.

## Creating an identity

To create an identity on the Radicle network, you can use the Upstream app. The app will generate a unique identifier — known as a [**Device ID**](understanding-radicle/glossary.md/#peerid) — that's paired to the computer you've installed Radicle on. For now, this Device ID is the only way other users can find and collaborate with you. Later on, you will be able to manage multiple Device IDs under one user handle, enabling support and consistency across multiple devices.

[insert pic of identity]

> The app will also prompt you to create a passphrase. This passphrase *isn't* a private key — it is a password decrypting your keypair to access and publish to the Upstream app. You will also be prompted for this passphrase when pushing code from your local machine to the Radicle network via your command line.

Reviewer 2: ```suggestion
The app will also prompt you to create a passphrase. This passphrase *isn't* a private key — it is a password decrypting your keypair to access and publish to the Upstream app. You will also be prompted for this passphrase when pushing and fetching code from your local machine to the Radicle network via your command line.
```

## Configuring your system

> To push code to the Radicle network via the Upstream client, you first have to add the [Radicle git remote helper](understanding-radicle/faq.md) to your shell configuration.

Reviewer 3: ```suggestion
To publish code to the Radicle network via the Upstream client, you first have to add the [Radicle git remote helper](understanding-radicle/faq.md) to your shell configuration.
```

<home path>/.radicle/bin/git-remote-rad
```

> With the remote helper installed, you will be able to [push changes](using-radicle/pushing-and-pulling-changes.md#pushing-changes-to-a-project) to the Radicle network.

Reviewer 3: ```suggestion
With the remote helper installed, you will be able to [push changes](using-radicle/pushing-and-pulling-changes.md#pushing-changes-to-a-project) to your Radicle installation, which will be picked up by the Upstream client.
```

Reviewer: What do you mean by Radicle installation?

Reviewer 3: I'm referring to the local radicle monorepo. Pushing itself doesn't result in propagation of the changes to the network, a peer-connected client like Upstream needs to pick it up and communicate.

Reviewer: Gotcha
